app/Controller/SongsController.php

```diff
@@ -244,32 +244,21 @@ public function albums() {
                 )
             );
         } else {
-            $subQuery = $db->buildStatement(
-                array(
-                    'fields' => array('MIN(subsong.id)', 'subsong.album'),
-                    'table' => $db->fullTableName($this->Song),
-                    'alias' => 'subsong',
-                    'group' => 'subsong.album'
-                ),
-                $this->Song
-            );
-            $subQuery = ' (Song.id, Song.album) IN (' . $subQuery . ') ';
-
             if ($page == 1) {
                 $latests = $this->Song->find('all', array(
-                    'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
-                    'conditions' => $subQuery,
+                    'fields' => array('MIN(Song.id)', 'Song.band', 'Song.album', 'Song.cover'),
                     'order' => 'Song.created DESC',
+                    'group' => 'Song.album',
```

Comment:
It seems I can't split them since the second commit does changes on what was introduced in the first commit? (getting conflicts) I guess the options then are 1) either remove my last commit, get the pull request merged by the owner and then re-commit and create the new pull request for the last commit 2) or let both commits stay in the same pull request.